### ISSUE TITLE
Figure out versioning strategy for vNext

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#19957]
Split from #19106

Details are in RFC #19658 but will be extracted into its own RFC for a long running discussion on how to version vNext packages.

* Independent versioning strategy
* Lockstep (full vs major lockstep)

**Comment by layershifter** [comment]
We have now #23914 that covers it. Should be this closed?

**Comment by ling1726** [comment]
I don't think this is the same actually, we still don't really know how we are going to do our first breaking change

**Comment by Hotell** [comment]
> I don't think this is the same actually, we still don't really know how we are going to do our first breaking change

IMO this issue should be named differently. if i understood it correctly we need a strategy how will we approach v10 per say.

regarding v9 as @layershifter mentioned we have #23914 .

WDYT @ling1726 ?

**Comment by ling1726** [comment]
@Hotell  is it incorrectly named ? this isn't about release but about versioning.
